gc/base/standard/RememberedSetWorkPackets.hpp

```diff
@@ -0,0 +1,93 @@
+/*******************************************************************************
+ * Copyright (c) 1991, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+
+#if !defined(REMEMBEREDSETWORKPACKETS_HPP_)
+#define REMEMBEREDSETWORKPACKETS_HPP_
+
+#include "j9.h"
+#include "j9cfg.h"
+#include "modron.h"
+#include "WorkPackets.hpp"
+#include "BaseNonVirtual.hpp"
+
+class EnvironmentModron;
+
+class MM_RememberedSetWorkPackets : public MM_BaseNonVirtual
+{
+/* Data members & types */
+public:
+	J9VMGCRememberedSet _rememberedSetStruct; /**< The VM-readable struct containing the remembered set "global" indexes. */
+protected:
+private:
+	MM_WorkPackets *_workPackets; /**< The workPackets struct used as backing store for the rememberedSet */

```

Comment:
this should point to more specific MM_WorkPacketsSATB class

Reply:
fixed. I also re-named RememberedSetWorkPackets.hpp to RememberedSetWorkPacketsSATB.hpp so it doesn't conflict with the one in openj9 (which will get removed after openj9 changes are in)